ui/src/app/views/workspace-edit/component.html

```diff
@@ -15,140 +15,167 @@ <h2 *ngSwitchCase="Mode.Clone">Clone workspace "{{oldWorkspaceName}}"</h2>
+                    for analyzing or interpreting data (e.g., developing more powerful methods to detect
+                    epistatic, gene-environment, or other types of complex interactions in genome-wide
+                    association studies). Data will be used for developing and/or validating new methods.
+                  </div>
+                </div>
               </div>
             </div>
             <div class="checkbox-group">
-              <clr-checkbox
-                  name="control-set"
-                  id="control-set"
-                  [(ngModel)]="workspace.researchPurpose.controlSet"
-                  [clrDisabled]="mode == Mode.Edit">
-                Control set
-              </clr-checkbox>
-              <div>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod bibendum laoreet. Proin gravida dolor sit amet lacus accumsan et viverra justo commodo.
+              <div style="display: flex">
+                <clr-checkbox
+                        name="aggregate-analysis"
+                        id="aggregate-analysis"
+                        [(ngModel)]="workspace.researchPurpose.aggregateAnalysis"
+                        [clrDisabled]="mode == Mode.Edit">
+                </clr-checkbox>
+                <div>
+                  <label class="checkbox-label" for="clr-checkbox-aggregate-analysis"><h3>Aggregate analysis to understand variation in general population</h3></label>
+                  <div>
+                    The primary purpose of the research is to understand variation in the
+                    general population (e.g., genetic substructure of a population).
+                  </div>
+                </div>
               </div>
             </div>
+          </div>
+          <div class="row">
             <div class="checkbox-group">
-              <clr-checkbox
-                  name="commercial-purpose"
-                  id="commercial-purpose"
-                  [(ngModel)]="workspace.researchPurpose.commercialPurpose"
-                  [clrDisabled]="mode == Mode.Edit">
-                Commercial purpose/entity
-              </clr-checkbox>
-              <div>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod bibendum laoreet. Proin gravida dolor sit amet lacus accumsan et viverra justo commodo.
+              <div style="display: flex">
+                <clr-checkbox
+                    name="control-set"
+                    id="control-set"
+                    [(ngModel)]="workspace.researchPurpose.controlSet"
+                    [clrDisabled]="mode == Mode.Edit">
+                </clr-checkbox>
+                <div>
+                  <label class="checkbox-label" for="clr-checkbox-control-set"><h3>Control Set</h3></label>
+                  <div>
+                    All of Us TM data will be used  increase the number of controls available for a
```

Comment:
TM? Trademark? Is this needed?

Reply:
The language here is directly from Moran, I can check with her on some of the details here re: e.g. and TM.

Reply:
its spelled `&trade;` though, not `TM` 😉